fastlane/Fastfile

```diff
@@ -25,8 +25,8 @@ platform :ios do
   app_secrets = ENV["APP_SECRETS"]
   app_secrets_path = "../Nos/Assets/ProductionSecrets.xcconfig"
 
-  desc "Push a new development build to TestFlight"
-  lane :dev do
+  desc "Push a new build to TestFlight"
```

Comment:
Maybe add something like this to provide a bit more detail?
```suggestion
  desc "Push a new build to TestFlight using the provided scheme and identifier"
```

I know there are other params in `options` but I'm not sure we need to specify all of them. Thoughts?